topics/basics/plugin_signing.md

```diff
@@ -0,0 +1,164 @@
+[//]: # (title: Plugin Signing)
+
+<!-- Copyright 2000-2021 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+
+Plugin Signing is a mechanism introduced in the 2021.2 release cycle to increase security in JetBrains Marketplace and all of our IntelliJ-based IDEs.
+
+The Marketplace signing is designed to ensure that plugins are not modified over the course of the publishing and delivery pipeline.
+If the plugin is not signed by the author or if a certificate has been revoked, a warning dialog will appear in the IDE during installation.
+
+On our side, we will check if the public part of a key is present, and we will verify the signature. This is similar to the [Google upload key](https://developer.android.com/studio/publish/app-signing#generate-key) mechanism.
+
```

Comment:
can we add some links to gentle introduction to the world of private/public keys, CAs, CRLs, etc.? some wikipedia?